surfsense_backend/app/agents/researcher/utils.py

```
@@ -1,14 +1,59 @@
 from typing import List, Dict, Any, Tuple, NamedTuple
 from langchain_core.messages import BaseMessage
+from pydantic import BaseModel, Field
 from litellm import token_counter, get_model_info
 
+class Section(BaseModel):
+    """A section in the answer outline."""
+    section_id: int = Field(..., description="The zero-based index of the section")
+    section_title: str = Field(..., description="The title of the section")
+    questions: List[str] = Field(..., description="Questions to research for this section")
+
+class AnswerOutline(BaseModel):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)